packages/core/src/plugins/index.ts

```diff
@@ -690,51 +692,78 @@ const _overridePlugin = (c: RnvContext, pluginsPath: string, dir: string) => {
             )}. skipping folder override action`
         );
     }
-
-    let overridePath: string | undefined;
-    if (plugin?.version) {
-        const pluginVerArr = plugin.version.split('.');
-        const pluginVersions: Array<string> = [];
-        let prevVersion: string;
-        pluginVerArr.forEach((v) => {
-            if (prevVersion) {
-                prevVersion = `${prevVersion}.${v}`;
-            } else {
-                prevVersion = `${v}`;
+    const overridePath = _findOverridePath(pluginsPath, dir, plugin);
+
+    _applyFileOverrides(overridePath, dest);
+
+    // override @react-native-community in nested packages
+    if (plugin?._id === '@react-native-community/cli' || plugin?._id === '@react-native-community/cli-platform-ios') {
```

Comment:
That is very specific. It would be better if we could look through packages and if they have their own node modules then look if there is anything from overrides that should be applied. Because it will solve the problem now, but only for this specific case, if anything comes up with android or there is node_modules not in react-native or react-native-tvos that need to be overridden it will not work